### ISSUE TITLE
SelectMany Dropdown don't load ajax content with Mysql

### ISSUE THREAD

**Issue by marcellovetro** [strapi/strapi#1953]
**Informations**
- **Node.js version**: v10.9.0
- **npm version**: 6.4.1
- **Strapi version**: 3.0.0-alpha.14.1.1
- **Database**: Mysql
- **Operating system**: Linux Ubuntu 16.04

**What is the current behavior?**
In Content Manager Plugin the SelectMany Dropdown that allow to add a relation don't start the ajax autofill when you write a key to search the related element.

**Steps to reproduce the problem**
1. Execute into your Mysql database the attached SQL that create "comments" table and import 500 record
[comments.sql.zip](https://github.com/strapi/strapi/files/2384234/comments.sql.zip)
2. create a ContentType with the following schema "comment" with collection name "comments"
```` 
{
  "connection": "default",
  "collectionName": "comments",
  "info": {
    "name": "comment",
    "description": "",
    "mainField": "name"
  },
  "options": {
    "increments": true,
    "timestamps": true,
    "comment": ""
  },
  "attributes": {
    "name": {
      "default": "",
      "type": "string"
    },
    "postId": {
      "default": "",
      "type": "integer"
    },

    "email": {
      "default": "",
      "type": "email"
    },
    "body": {
      "default": "",
      "type": "text"
    },
    "linked": {
      "collection": "comment",
      "via": "links"
    },
    "links": {
      "model": "comment",
      "via": "linked"
    }
  }
}
````
3. Open in the admin the Comments content and try to search in both the "Linked" and "Links" relation the key "doloribus dolores ut dolores"
4. You can see that the SelectOne "Links" works good: It show the record, instead the SelectMany work bad responsing "No result found"

**What is the expected behavior?**
Both "Linked" and "Links" relation would show the record

**Suggested solutions**
Check SelectMany component and start the ajax search where you type a key in the dropdown, like the OneToMany Component

**Comment by derrickmehaffy** [comment]
@marcellovetro I'm confused why are you creating two relations to the same model? with different types?

**Comment by marcellovetro** [comment]
Only to use a single table and make easy reproduce the problem, the SelectMany work bad also if you have two different table!!!

**Comment by marcellovetro** [comment]
Fixed in PR  #1963
